src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -67,11 +76,27 @@ export const CreateOAuthModal: React.FunctionComponent<
     onClose()
   }
 
+  const hideConfirmModal = () => {
+    setIsShowingConfirmModal(false)
+    setIsDelete(false)
+    setSelectedClient(undefined)
+  }
+
   const { mutate } = useMutateOAuthClient({
     onSuccess: () => {
       displayToast(`Successfully saved`, 'success')
       hide()
     },
+    onError: (err: any) => {
+      displayToast(err.reason as string, 'danger')
+    },
+  })
+
+  const deleteClient = useDeleteOAuthClient({
```

Comment:
When using multiple mutations, I've started doing something like
```suggestion
  const { mutate: delete } = useDeleteOAuthClient({
```

Then instead of `deleteClient.mutate(...)` you could do `delete(...)`. I think it's more readable but I'll leave it up to you.